project.md

```diff
@@ -1,33 +1,20 @@
-# Project Idea Pitch
-
-Edit this file to include your project idea pitch for the HighTechU Academy.
-
 ## Project Title
 
-[DELETE ME - ADD YOUR PROJECT TITLE HERE]
-
-Example - Colour Picker
+B.I.G.
+Books Incur Genius
 
 ## Project Mini Pitch
 
-[DELETE ME - ADD YOUR PROJECT MINI PITCH HERE]
-
-Example - This colour picker will revolutionaize the way colours are picked for the web.
+Teenagers don't read enough, but by supplying an easier access to material, social connections, and relevant reviews we hope to change that.
 
 ## Project Description
 
-[DELETE ME - ADD YOUR PROJECT DESCRIPTION HERE]
-
-Example - The app will be web based and allow users to pick a colour with the help of a colour wheel.
+The app is designed to facilitate reading for teenagers by creating profiles and sorting relevant reviews based on: friends on other social platforms, previous reads, personal interests, and more.
 
 ## Project Audience
 
-[DELETE ME - ADD YOUR PROJECT AUDIENCE HERE]
-
-Example - The audience would be frontend developers and designers looking to pick colours for their work.
+This app is designed for use by teenagers on Vancouver island, but as it develops it could be expanded to include other areas and ages.
 
 ## Project Inspiration
 
-[DELETE ME - ADD YOUR PROJECT INSPIRATION HERE]
-
-Example - This project idea was inspired by the following websites: list of website.
```

Comment:
Hey Jamie!

The HighTechU Staff Team love this project idea! We can't wait to see it!